Task8.py

```diff
@@ -10,4 +10,4 @@
 m = int(input('длина плитки в дольках: '))
 k = int(input('количество долек, которое нужно отломить за один разлом: '))
 
-print(k <= m * n and k % n == 0 or k % m == 0)
+print((1 <= k < n * m) and (k % n == 0 or k % m == 0))
```

Comment:
Верно, но первые скобки нужны
```suggestion
print(1 <= k < n * m and (k % n == 0 or k % m == 0))
```